ts/features/bonus/cgn/screens/eyca/activation/EycaActivationLoading.tsx

```diff
@@ -31,11 +28,12 @@ const EycaActivationLoading: React.FunctionComponent<Props> = (
   />
 );
 
-const mapStateToProps = (state: GlobalState) => ({
-  isLoading:
-    cgnEycaActivationLoading(state) ||
-    fromNullable(cgnEycaActivationStatus(state)).fold(true, s => s !== "ERROR")
-});
+const mapStateToProps = (state: GlobalState) => {
+  const eycaActivation = eycaActivationStatusSelector(state);
+  return {
+    isLoading: isLoading(eycaActivation) || !isError(eycaActivation)
```

Comment:
Why do you need also `|| !isError(eycaActivation)`?
it is not enough `isLoading(eycaActivation)`?

Reply:
We may be in this screen even if we are loading the result and we have a value of this type: 
```typescript
{
  king: "ready",
  status: "POLLING"
} 
```

in this case even if value is not in an error state we would show the error and retry component